samples/StaticHtmlComposites/src/composites.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  CallComposite,
+  createAzureCommunicationCallAdapter,
+  ChatComposite,
+  createAzureCommunicationChatAdapter
+} from '@azure/communication-react';
+
+export const loadCallComposite = async function (args) {
+  const { containerId, userId, token, groupId, displayName } = args;
+  const adapter = await createAzureCommunicationCallAdapter(userId, token, { groupId }, displayName ?? 'anonymous');
+  ReactDOM.render(React.createElement(CallComposite, { adapter }, null), document.getElementById(containerId));
+  return adapter;
+};
+
+export const loadChatComposite = async function (args) {
+  const { containerId, userId, token, endpointUrl, threadId, displayName } = args;
+  const adapter = await createAzureCommunicationChatAdapter(
+    userId,
```

Comment:
The order here is easy to get wrong. We should change the API to use objects if there are more than 2 arguments in a function. 

```typescript
createAzureCommunicationChatAdapter({
   userId: ...,
   token: ...,
   ...
 });